app/alerts.py

```diff
@@ -131,6 +153,52 @@ def build_sites_markers(dpt_code=None):
     return dl.MarkerClusterGroup(children=markers, id='sites_markers')
 
 
+def build_vision_polygon(site_lat, site_lon, yaw, opening_angle, dist_km):
+    """
+    This function allows to build the vision angle of a camera, ie. the zone covered by the detection device.
+
+    It takes as input:
+
+    - site_lat, the latitude of the detection device;
```

Comment:
Nice function !! 

Quick question: will we have all these inputs through the API ?